pkg/kubelet/pod_workers.go

```diff
@@ -39,6 +40,14 @@ type PodWorkers interface {
 
 type syncPodFnType func(*api.Pod, *api.Pod, *kubecontainer.PodStatus, kubetypes.SyncPodType) error
 
+const (
```

Comment:
Can we use a smaller factor such as 0.5? I think that should be enough to distribute the sync times.